support-frontend/assets/components/checkoutNudge/checkoutNudge.tsx

```diff
@@ -61,76 +58,77 @@ const headlineOverrides = css`
 
 const bodyCopyOverrides = css`
 	margin-top: ${space[2]}px;
-	${textSans15}
+	${textSans15};
 	font-weight: 700;
 `;
 
 const nudgeButtonOverrides = css`
 	margin-top: ${space[2]}px;
-	${textSans17}
+	${textSans17};
 	font-weight: 700;
 	width: 100%;
 `;
 
+export interface CheckoutNudgeProps {
+	supportRegionId: SupportRegionId;
+	heading: string;
+	body?: string;
+	product: ActiveProductKey;
+	ratePlan: ActiveRatePlanKey;
+	amount: number;
+}
+
 export function CheckoutNudge({
 	supportRegionId,
-	ratePlanKey,
-	recurringAmount,
-	abTestName,
-	abTestVariant,
+	heading,
+	body,
+	product,
+	ratePlan,
+	amount,
 }: CheckoutNudgeProps) {
 	useEffect(() => {
-		trackComponentLoad(`checkoutNudge-${abTestName}--${abTestVariant}`);
+		trackComponentLoad('checkoutNudge');
 	}, []);
 
 	const { currency } = getSupportRegionIdConfig(supportRegionId);
 
-	if (!['Monthly', 'Annual'].includes(ratePlanKey)) {
-		ratePlanKey = 'Monthly';
+	if (!(ratePlan === 'Monthly' || ratePlan === 'Annual')) {
+		// Only these rate plans are currently supported
+		return null;
 	}
+	const ratePlanDescription = ratePlan === 'Monthly' ? 'month' : 'year';
 
-	const ratePlanDescription = ratePlanKey === 'Monthly' ? 'month' : 'year';
-
+	// TODO - different copy?
```

Comment:
When we enable the nudge for more products the CTA copy will need to change.
Arguably the CTA copy could also come from the config. But this is fiddly as it will need templating for the amount/currency/frequency.
I think for now we should hardcode the copy here based on the product.